test-clients/src/main/java/com/hedera/services/bdd/suites/schedule/ScheduleDeleteSpecs.java

```diff
@@ -42,7 +53,30 @@ protected Logger getResultsLogger() {
     @Override
     protected List<HapiApiSpec> getSpecsInSuite() {
         return List.of(new HapiApiSpec[] {
-                }
-        );
+                expiredBeforeSigning(),
+        });
+    }
+
+    public HapiApiSpec expiredBeforeSigning() {
+        final int FAST_EXPIRATION = 0;
+        return defaultHapiSpec("SignFailsDueToDeletedSignDueToExpiration")
+                .given(
+                        cryptoCreate("sender").balance(1L),
+                        cryptoCreate("receiver").balance(0L).receiverSigRequired(true)
+                ).when(
+                        fileUpdate(APP_PROPERTIES)
+                                .payingWith(ADDRESS_BOOK_CONTROL)
+                                .overridingProps(Map.of("ledger.schedule.txExpiryTimeSecs", "" + FAST_EXPIRATION)),
+                        scheduleCreate(
+                                "twoSigXfer",
+                                cryptoTransfer(
+                                        tinyBarsFromTo("sender", "receiver", 1)
+                                ).fee(ONE_HBAR).signedBy("sender")
+                        ).logged(),
```

Comment:
I think that you will need to do `inheritingScheduledSigs` to add the `sender`s signature to the `schedule create` sig map. This is changed based on the latest PR that was merged into `schedule-develop`